src/Microsoft.AspNet.Authentication.Facebook/FacebookAppBuilderExtensions.cs

```diff
@@ -18,10 +17,22 @@ public static class FacebookAppBuilderExtensions
         /// </summary>
         /// <param name="app">The <see cref="IApplicationBuilder"/> passed to the configure method.</param>
         /// <returns>The updated <see cref="IApplicationBuilder"/>.</returns>
-        public static IApplicationBuilder UseFacebookAuthentication([NotNull] this IApplicationBuilder app, Action<FacebookOptions> configureOptions = null)
+        public static IApplicationBuilder UseFacebookAuthentication([NotNull] this IApplicationBuilder app, FacebookOptions options)
```

Comment:
The `options` parameter is not guarded against null values (nor decorated with `NotNull`) but `FacebookMiddleware`'s `options` parameter has `NotNull` applied. Is it worth making everything consistent?
The same remark seems to apply to all the authentication middleware.